pkgs/applications/networking/cluster/nomad/1.0.nix

```diff
@@ -0,0 +1,14 @@
+{ callPackage, buildGoPackage, fetchpatch }:
+
+callPackage ./generic.nix {
+  inherit buildGoPackage;
+  version = "1.0.0";
+  sha256 = "04aa8jjjfykj901rqvfb9l9jimk8k0vciznb3m2jx60b4yd7mjqc";
+  patches = [
+    # without this patch the -var-file argument is not passed in to `nomad job run`
```

Comment:
I have complicated feelings about packages including patches - given the history of the Nomad team I assume a release will be out next week with this included (plus any other early bugs that were found). I'll ask around.

Reply:
Fair enough. I don't feel so strongly about this though I would hate for this to land and people start using it only to discover they can't use `-var-file` because it's missing. OTOH, I am also fine with waiting.

Reply:
@endocrimes Were you able to ask around about the next release? I am also happy to ping folks over on the nomad repo as well.

Reply:
Looks like a new release is out https://github.com/hashicorp/nomad/releases/tag/v1.0.1

Reply:
And some bugs fixed to boot! I'll update this PR.